Containerfile.rag

```diff
@@ -1,14 +1,2 @@
 # vim: set filetype=dockerfile
-ARG OLS_VERSION=latest
-ARG LIGHTSPEED_RAG_CONTENT_DIGEST=sha256:d15bf56776c40a8709b0e648e3b0f043de63b24ad8f59eeea6f8d965dfcbe4e3
-
-FROM quay.io/openshift/lightspeed-rag-content@${LIGHTSPEED_RAG_CONTENT_DIGEST} as lightspeed-rag-content
-
 FROM quay.io/openshift/lightspeed-service-api:${OLS_VERSION}
```

Comment:
How does get an OLS_VERSION? Are we defaulting to empty therefore "latest" here?

Reply:
it's a good catch that i should have left the ARG in place to default to latest, but in reality the whole FROM line is getting overwritten by openshift CI.

fixed, though it doesn't matter that much because this entire file is going to be deleted very soon.